addons/binding/org.openhab.binding.co2signal/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="co2signal" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- CO2 Signal Thing -->
+	<thing-type id="co2signal">
+		<label>CO2 Signal</label>
+		<description>
+		Provides information about the CO2 intensity of produced electricity of a given location.
+		In order to receive the data, you must register an account on https://www.co2signal.com/ and get your API token.
+        </description>
+
+		<channels>
+            <channel id="countryCode" typeId="countryCode" />
```

Comment:
XML files should be indented using tabs according to our code formatter, this one looks mixed

Reply:
Fixed but wondering why this isn't enforced in other bindings. I have yet to find a binding where all xml files are formatted the right way